src/components/LagPopover/LagPopover.scss

```diff
@@ -0,0 +1,13 @@
+.ydb-lag-popover {
+    div:nth-child(1) {
```

Comment:
Let's define css properties only with classes, not with html-tags, it's not a BEM-way.